lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -19,6 +20,11 @@
                 el: '.sidebar',
                 context: options
             }).render();
+
+            new CertificateView(
+                options.certificatesData
+            ).render();
```

Comment:
Please rebase and update to instantiate the CertificateView in the postRender function of the SidebarView (https://github.com/edx/edx-platform/blob/master/lms/static/js/learner_dashboard/views/sidebar_view.js#L33)